notebooks/summit/AuxTel/SITCOM-627/SITCOM-627_Measure_Astigmatism.ipynb

```diff
@@ -0,0 +1,1511 @@
+{
```

Comment:
<p>Is 30s needed here? Seems long for an acquisition?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Wanted to use the same as the exposure to check also signal levels and varied star magnitudes. Probably shouldn't be called acquisition.

Reply:
Will call it "Test Exposure"